pre_push.py

```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env python3
 """Run static analysis on the project."""
+
+from os import path
```

Comment:
Please move this to after `import sys` as these are all system packages.

Reply:
I did that.